com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/RTUpscale.hlsl

```diff
@@ -82,9 +81,13 @@ CTYPE Lanczos(TEXTURE2D_X(_InputTexture), float2 inUV)
     float2 xMin4 = x - 4.0;
     float2 xMin5 = x - 5.0;
 
+    // By the definition of x, only xMin2 should possibly = 0
+    xMin2 = (xMin2 == 0) * epsilon + xMin2; // if (xMin2 == 0), then xMin2 = epsilon
```

Comment:
won't this be simpler (and more efficient) to always do:

float2 xMin2 = x - 2.0 + epsilon;

epsilon is negligible anyway.

Reply:
ok forget it, don't solve the issue